Vialer/AccountViewController.m

```diff
@@ -0,0 +1,154 @@
+//
+//  AccountViewController.m
+//  Vialer
+//
+//  Created by Harold on 18/06/15.
+//  Copyright (c) 2015 VoIPGRID. All rights reserved.
+//
+
+#import "AccountViewController.h"
+#import "VoIPGRIDRequestOperationManager.h"
+#import "AccountViewFooterView.h"
+#import "EditNumberTableViewController.h"
+
+@interface AccountViewController ()
+@end
+
+@implementation AccountViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 2; //3 if you want to display logout
+}
+//To enable the logout button,
+//- change number of sections to 3
+//- change LOGOUT_BUTTON_SECTION to 1
+//- change NUMBERS_SECTION to 2
+
+#define VOIP_ACCOUNT_SECTION 0
+#define SIP_ACCOUNT_ROW 0
+#define SIP_PASSWORD_ROW 1
+
+#define LOGOUT_BUTTON_SECTION 99 //unused should be 1
+#define LOGOUT_BUTTON_ROW 0
+
+#define NUMBERS_SECTION 1 //was 2
+#define MY_NUMBER_ROW 0
+#define OUTGOING_NUMBER_ROW 1
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    if (section == LOGOUT_BUTTON_SECTION)
+        return 1;
+    return 2;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    static NSString *CellIdentifier = @"ConfigViewCell";
+    UITableViewCell *cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleValue1 reuseIdentifier:CellIdentifier];
```

Comment:
TableView has a method called dequeueCellWithIdentifier:indexPath: this uses reusable cells and is more efficient than allocating every time.